sympy/stats/tests/test_finite_rv.py

```diff
@@ -122,6 +122,7 @@ def test_domains():
     Z = Die('x', 4)
 
     raises(ValueError, lambda: P(X > Z))  # Two domains with same internal symbol
+    raises(ValueError, lambda: P(X > y))  # Tests for foreign symbol
```

Comment:
What happens if you just pass a symbol? Consider that at the time `P` was introduced, it was supposed to always evaluate the expression. Now we have `Probability` and we may return unevaluated expressions. I think it would be preferrable to do so.

Reply:
Yes, I am trying to work on it. I have two approaches in my mind. One I will push and the other one I will explain in comments.